.envs/.production/.django

```diff
@@ -48,16 +48,31 @@ CELERY_FLOWER_USER=kJtrEYByunMyFNPJeALMRoyxcRTInfFw
 CELERY_FLOWER_PASSWORD=dz3RyDpw47ViMIcAbwdsfwZEPmrAWl5zEa5boiJs3WAN7cFaR60hkNNvxlwqQd2d
 
 # Jira
-# JIRA_SERVER=https://jira.server
-# JIRA_USERNAME=username
-# JIRA_PASSWORD=password
+JIRA_SERVER=https://jira.server
+JIRA_USERNAME=
+JIRA_PASSWORD=
+JIRA_CELL_ROLE_ACCOUNT=
+JIRA_CELL_ROLES=
 
 # Google
-# GOOGLE_API_PRIVATE_KEY=private_key
-# GOOGLE_API_CLIENT_EMAIL=client_email
-# GOOGLE_SPILLOVER_SPREADSHEET=spreadsheet_id
+GOOGLE_API_PRIVATE_KEY=
+GOOGLE_API_CLIENT_EMAIL=
+GOOGLE_SPILLOVER_SPREADSHEET=
+GOOGLE_CONTACT_SPREADSHEET=
+GOOGLE_AVAILABILITY_RANGE=
+GOOGLE_ROTATIONS_SPREADSHEET=
+GOOGLE_ROTATIONS_RANGE=
 
 # JWT
-# DJANGO_JWT_EXPIRATION_MINUTES=5
-# DJANGO_JWT_ALLOW_REFRESH=True
-# DJANGO_JWT_REFRESH_EXPIRATION_DAYS=7
+# JWT_ACCESS_TOKEN_LIFETIME_HOURS=3
+# JWT_REFRESH_TOKEN_LIFETIME_DAYS=30
+
+CELERY_BROKER_URL=redis://redis:6379/0
+DATABASE_URL=postgres://WmBQECsnGEnJEoysAOQIgHDHpxnNIKYt:RWw3Oc07aYWXou8chyzm61WwVXxrC0Cht0Nkg9VXOjHxtyTHpzDuTw89PxzPR5cv@postgres:5432/sprints
```

Comment:
@Agrendalath Similar comment here, if these are production secrets then we should remove them?

Reply:
These are just some random values that can help spawning prod environment locally without too much configuration changes. They're definitely not used on the prod :)

Reply:
I renamed them to make this less confusing.

Reply:
@Agrendalath Sounds good, thanks!